[FEATURE] The event 'CancelTransfer' was not found in this contract's abi.

## Is your feature request related to a problem? Please describe.
The following error is occurring in an environment that handles old format tokens.

```
[2023-08-28 04:39:30 +0000] [156] [ERROR] [PROCESSOR-NOTIFICATIONS-TOKEN] ("The event 'CancelTransfer' was not found in this contract's abi. ", 'Are you sure you provided the correct contract abi?')
[2023-08-28 04:40:29 +0000] [156] [ERROR] [PROCESSOR-NOTIFICATIONS-TOKEN] ("The event 'ApproveTransfer' was not found in this contract's abi. ", 'Are you sure you provided the correct contract abi?')
```

We won't support the full functionality of tokens in the old format, but I think it's a problem if the data is present and an error occurs.

## Describe the solution you'd like
I would like to add error handling to prevent errors from occurring.

--- a/batch/processor_Notifications_Coupon_Exchange.py
+++ b/batch/processor_Notifications_Coupon_Exchange.py
@@ -25,6 +25,7 @@ from datetime import datetime
 from sqlalchemy import create_engine, select
 from sqlalchemy.exc import SQLAlchemyError
 from sqlalchemy.orm import Session
+from web3.exceptions import ABIEventFunctionNotFound
 
 path = os.path.join(os.path.dirname(__file__), "../")
 sys.path.append(path)
@@ -128,10 +129,13 @@ class Watcher:
                 to_block_number = latest_block_number
 
             # Get event logs
-            _event = getattr(self.contract.events, self.filter_name)
-            entries = _event.get_logs(
-                fromBlock=from_block_number, toBlock=to_block_number
-            )
+            try:
+                _event = getattr(self.contract.events, self.filter_name)
+                entries = _event.get_logs(
+                    fromBlock=from_block_number, toBlock=to_block_number
+                )
+            except ABIEventFunctionNotFound:
+                entries = []
 
             # Register notifications
             if len(entries) > 0:

--- a/batch/processor_Notifications_Membership_Exchange.py
+++ b/batch/processor_Notifications_Membership_Exchange.py
@@ -25,6 +25,7 @@ from datetime import datetime
 from sqlalchemy import create_engine, select
 from sqlalchemy.exc import SQLAlchemyError
 from sqlalchemy.orm import Session
+from web3.exceptions import ABIEventFunctionNotFound
 
 path = os.path.join(os.path.dirname(__file__), "../")
 sys.path.append(path)
@@ -128,10 +129,13 @@ class Watcher:
                 to_block_number = latest_block_number
 
             # Get event logs
-            _event = getattr(self.contract.events, self.filter_name)
-            entries = _event.get_logs(
-                fromBlock=from_block_number, toBlock=to_block_number
-            )
+            try:
+                _event = getattr(self.contract.events, self.filter_name)
+                entries = _event.get_logs(
+                    fromBlock=from_block_number, toBlock=to_block_number
+                )
+            except ABIEventFunctionNotFound:
+                entries = []
 
             # Register notifications
             if len(entries) > 0:

--- a/batch/processor_Notifications_Token.py
+++ b/batch/processor_Notifications_Token.py
@@ -27,7 +27,7 @@ from sqlalchemy import and_, create_engine, select
 from sqlalchemy.exc import SQLAlchemyError
 from sqlalchemy.orm import Session
 from web3.contract import Contract as Web3Contract
-from web3.exceptions import MismatchedABI
+from web3.exceptions import ABIEventFunctionNotFound
 from web3.types import EventData
 
 path = os.path.join(os.path.dirname(__file__), "../")
@@ -171,9 +171,9 @@ class Watcher:
                     entries = _event.get_logs(
                         fromBlock=from_block_number, toBlock=to_block_number
                     )
+                except ABIEventFunctionNotFound:  # Backward compatibility
+                    entries = []
                 except FileNotFoundError:
-                    continue
-                except MismatchedABI:  # Backward compatibility
                     continue
                 except Exception as err:  # If an Exception occurs, processing continues
                     LOG.error(err)

--- a/batch/processor_Notifications_Token.py
+++ b/batch/processor_Notifications_Token.py
@@ -27,6 +27,7 @@ from sqlalchemy import and_, create_engine, select
 from sqlalchemy.exc import SQLAlchemyError
 from sqlalchemy.orm import Session
 from web3.contract import Contract as Web3Contract
+from web3.exceptions import MismatchedABI
 from web3.types import EventData
 
 path = os.path.join(os.path.dirname(__file__), "../")
@@ -171,6 +172,8 @@ class Watcher:
                         fromBlock=from_block_number, toBlock=to_block_number
                     )
                 except FileNotFoundError:
+                    continue
+                except MismatchedABI:  # Backward compatibility
                     continue
                 except Exception as err:  # If an Exception occurs, processing continues
                     LOG.error(err)

